src/components/Calendar/Calendar.js

```diff
@@ -65,23 +86,54 @@ const MyCalendar = props => {
     })
     setShowCalendar(!showCalendar)
   }
+
+
+
   const handleEventClick = e => {
-    setDragSelected({
-      ...dragSelected,
-      start: e.start,
-      end: e.end,
-      mentor: e.mentor,
-      student: e.student,
-      resourceId: e.resourceId
+
+    //If in week view clicking an event will hide the calendar, to show computer list
+
+    if(showWeekView){
+      setDragSelected({
+        ...dragSelected,
+        start: e.start,
+        end: e.end,
+        mentor: e.mentor,
+        student: e.student,
+        resourceId: e.resourceId
+      }
+      );
+      setShowCalendar(!showCalendar)
+    }
+
+    // Else clicking a event will bring pop-up of event details
+
+    else {
+      setClickSelected({
+        ...dragSelected,
+        start: e.start,
+        end: e.end,
+        mentor: e.mentor,
+        student: e.student,
+        resourceId: e.resourceId,
+        title: e.title
+      })
+      console.log(e)
```

Comment:
Can you remove this console log before merging?

Reply:
Also, removed this one. Thanks!